shell/components/form/__tests__/LabeledSelect.test.ts

```diff
@@ -1,5 +1,5 @@
 import { mount } from '@vue/test-utils';
-import LabeledSelect from '@shell/components/form/LabeledSelect.vue';
+import LabeledSelect from '@shell/components/form/LabeledSelect/index.vue';
```

Comment:
same here

Reply:
Same as https://github.com/rancher/dashboard/pull/10068#discussion_r1474680928 (but this is a TS file)